### PR TITLE
n-triple data import tool

### PR DESCRIPTION
**Pull request by transcranial**
re: issue #9

### CODE REVIEW

**Comment by mcollina** [comment]
@elf-pavlik what do you think? 

I will merge this and add @transcranial as a contributor, it it is ok for you too!

**Comment by elf-pavlik** [comment]
@transcranial can you think of some automated tests you could add to this PR?

**Comment by transcranial** [comment]
@elf-pavlik wrote some tests, please take a look!

**Comment by elf-pavlik** [comment]
Thanks!
@mcollina looks good for me to merge and yes please add @transcranial as contributor :)

File: import.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+  });
+
+} catch (err) {
+  console.error(err);
+  process.exit(1);
+} finally {
+  if (!options.quiet) console.log('\nAll files imported to levelgraph-n3 database at: ' + dbPath + '\n');
+}
```
Comment (mcollina) [inline import.js]: This finally block is not going to work, as you should listen for the end of the import stream.

I recommend the use of [eos](http://npm.im/eos) to detect it.

**Comment by mcollina** [comment]
Once that little fix is solved, @elf-pavlik feel free to merge this and release it as 0.5.0 :).

**Comment by elf-pavlik** [comment]
@mcollina why do we need to bump version from 0.4.0 to 0.5.0 ?

**Comment by mcollina** [comment]
It's a new feature, so a minor bump is ok :). Patch releases are for bugfixes.

**Comment by transcranial** [comment]
@elf-pavlik @mcollina thanks for the catch! Fixed.

**Comment by mcollina** [comment]
Should we also bump n3 to 0.4.x?

**Comment by elf-pavlik** [comment]
published v0.5.0 to NPM! thanks a lot @transcranial :+1:

**Pull request merged**